### PR TITLE
Feature/ons 19 multiple layer control

### PR DESCRIPTION
**Pull request by ashleighho-kainos**

### CODE REVIEW

File: map-demo/app/assets/javascripts/map.js [hunk 1]
```diff
@@ -1,40 +1,56 @@
...
-    .setLatLng(e.latlng)
-    .setContent("You clicked the map at " + e.latlng.toString())
-    .openOn(mymap);
-}
-
-mymap.on('click', onMapClick);*/
+var streets = L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token=pk.eyJ1IjoiYXNobGVpZ2hobyIsImEiOiJjaXN6d3l6MjUwMDYxMnlvM2djajZudG1zIn0.hAdrsz6R7-4Z9OepbVmMOA',
+                          {id: 'ashleighho.1ccd1a7b',
+                           attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="http://mapbox.com">Mapbox</a>',
+
```
Comment (lukem-kainos) [inline map-demo/app/assets/javascripts/map.js]: whitespace and formatting.

File: map-demo/app/assets/javascripts/map.js [hunk 2]
```diff
@@ -1,40 +1,56 @@
...
+                            }),
+    light     = L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token=pk.eyJ1IjoiYXNobGVpZ2hobyIsImEiOiJjaXN6d3l6MjUwMDYxMnlvM2djajZudG1zIn0.hAdrsz6R7-4Z9OepbVmMOA',
+                            {id: 'ashleighho.1d82ma2b',
+                              attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="http://mapbox.com">Mapbox</a>',
+
```
Comment (lukem-kainos) [inline map-demo/app/assets/javascripts/map.js]: same whitespace and formatting.

File: map-demo/app/views/map.scala.html [hunk 3]
```diff
@@ -14,6 +14,32 @@
             </p>
         </div>
 
+        <h4 class="heading-small">Layer controls</h4>
+        <div>
+            <form>
+                <fieldset class="inline">
+
```
Comment (lukem-kainos) [inline map-demo/app/views/map.scala.html]: whitespace

File: map-demo/app/views/map.scala.html [hunk 4]
```diff
@@ -14,6 +14,32 @@
...
+                        <input id="economy" name="layer-toggles" type="checkbox" value="economy">
+                        Economy
+                    </label>
+                    <label class="block-label" for="housing">
+                        <input id="housing" name="layer-toggles" type="checkbox" value="housing">
+                        Housing
+                    </label>
+                </fieldset>
+            </form>
+
```
Comment (lukem-kainos) [inline map-demo/app/views/map.scala.html]: whitespace

**Review by lukem-kainos — CHANGES_REQUESTED** [review]

File: map-demo/app/assets/javascripts/map.js [hunk 5]
```diff
@@ -1,40 +1,56 @@
...
+                            });
+
```
Comment (sdonaghey03) [inline map-demo/app/assets/javascripts/map.js]: Indentation and formatting could do with a touch up here

File: map-demo/app/views/map.scala.html [hunk 6]
```diff
@@ -14,6 +14,32 @@
...
+        </div>
+
```
Comment (sdonaghey03) [inline map-demo/app/views/map.scala.html]: Do we still need this form if it's built in to the map?

**Review by timja — APPROVED** [review]

**Review by lukem-kainos — APPROVED** [review]

**Pull request merged**